api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -18,6 +18,12 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Wasm]
 // [#extension: envoy.bootstrap.wasm]
 
+// Configuration for restricting proxy-wasm capabilities available to modules.
+message CapabilityRestrictionConfig {
+  // The list of proxy-wasm capabilities which will be exposed to the module.
+  repeated string allowed_abi_functions = 1;
```

Comment:
Similarly to the proxy-wasm-cpp-host comment, I think this should be `allowe_capabilities` or similar.

Reply:
Fixed.